spec/migrations/encryptables_data_structure_spec.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+mig_file = Dir[Rails.root.join('db/migrate/20220104140658_use_encrypted_data_for_account_credentials.rb')].first
+require mig_file
+
+describe UseEncryptedDataForAccountCredentials do
+
+  let(:migration) { described_class.new }
+
+  let(:folder1) {folders(:folder1)}
+
+  let(:account1) {accounts(:account1)}
+  let(:account2) {accounts(:account2)}
+  let!(:account3) do
+    Account::Credentials.create!(name: 'spacex', folder: folder1, encrypted_data: {
+      password: {data: '', iv: nil},
+      username: {data: nil, iv: nil}
+    })
+  end
+
+
+  def silent
+    verbose = ActiveRecord::Migration.verbose = false
+
+    yield
+
+    ActiveRecord::Migration.verbose = verbose
+  end
+
+  around do |test|
+    silent { test.run }
+  end
+
+  context 'up' do
+    before { migration.down }
```

Comment:
create LegacyAccount::Credentials in here with blank values for :username, :password